app/javascript/controllers/preview_filmstrip_controller.js

```diff
@@ -36,8 +35,8 @@ export default class extends Controller {
     this.preview.addClass('preview').empty();
 
     this.appendPointer()
-
-    PreviewContent.append(this.urlValue, divContent);
+    const id = this.urlValue.split('/').pop()
```

Comment:
I think we probably hate this, right? Is it possible to either pass the id in somewhere or not have to dynamically generate the HTML?

Reply:
Yes. I've passed it in.